ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/helpers/MathHelpers.java

```diff
@@ -48,6 +48,10 @@ public static Bytes uintToBytes(long value, int numBytes) {
     }
   }
 
+  public static Bytes uint64ToBytes(long value) {
```

Comment:
nit: Would it make more sense for this to take `UInt64` and call `.longValue()` itself?

Reply:
yeah - we could probably just have 2 versions